modules/nw-networkpolicy-deny-all-allowed.adoc

```diff
@@ -46,7 +46,7 @@ metadata:
   name: deny-by-default
   namespace: my-project <1>
   annotations:
-    k8s.v1.cni.cncf.io/policy-for: <namespace_name>/<network_name> <2>
+    k8s.v1.cni.cncf.io/policy-for: <network_name> <2>
```

Comment:
I wouldn't remove the namespace name from here: it will then default to whatever project the user has configured in `oc`. I've seen a lot of errors originating from that.